### PR TITLE
#4928 - Show zero price in my orders table

### PR DESCRIPTION
**Pull request by IrinaZhadzinets**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/4928

**Problem:**
* Price 'Total' absent for 0$ price order in 'My orders'

**In this PR:**
* Show 0$ price for total in 'My orders' table

### CODE REVIEW

**Comment by Elena-Evsa** [comment]
Can not check new orders, but for old orders - fixed

File: packages/scandipwa/src/component/MyAccountOrderTableRow/MyAccountOrderTableRow.component.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class MyAccountOrderTableRow extends PureComponent {
                 <td>{ created_at }</td>
                 <td>{ status }</td>
                 <td block="hidden-mobile">
-                    { value ? formatPrice(value, currency) : '' }
+                    { typeof value === 'number' ? formatPrice(value, currency) : '' }
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/MyAccountOrderTableRow/MyAccountOrderTableRow.component.js:49]: Please show 0 for all falsy or empty values as well.

Comment (IrinaZhadzinets) [reply packages/scandipwa/src/component/MyAccountOrderTableRow/MyAccountOrderTableRow.component.js:49]: If it is about ' ', then it is displayed only when loading, i.e. if I put some value there, the entire column will be filled with zeros when there is no other data yet

Comment (zans-laksa) [reply packages/scandipwa/src/component/MyAccountOrderTableRow/MyAccountOrderTableRow.component.js:49]: But can we detect that loading state and not render anything during the loading?
Now it's an implicit feature and we will forget how it works after a month and we might break something.

**Review by zans-laksa — APPROVED** [review]

**Pull request merged**